Penumbra/Penumbra.cs

```diff
@@ -91,6 +95,8 @@ public Penumbra( DalamudPluginInterface pluginInterface )
             PluginLog.Debug( "Triggered Redraw of {Player}.", p.Name );
             ObjectReloader.RedrawObject( p, RedrawType.OnlyWithSettings );
         };
+
+        _collections = _modManager.Collections.Collections.Values.ToArray(); 
```

Comment:
I fixed and committed it myself already, but what was your rationale behind this?
You're creating a copy with the collections available during start that doesn't get updated with new collections ever, instead of just using the available dictionary directly for linq. (I get that you're not using the dictionary accessor directly due to case-sensitivity).

Reply:
I originally planned on appending a "None" collection to the array, as well as having it be updated when new collections were added but I wanted to get the command functional first. I ended up just handling "None" as an edge case in the code itself, so I should have gone back and removed the _collections array but it slipped my mind. If I submit any more code I'll be sure to be more thorough in reviewing my own code aside from "well, it seems to work."

Reply:
No worries. I'm not sure if I should rather do the fixes in such a case myself (faster, less work) or just make the comments. You can look at my other commit for a few further things I've changed (and something that you completely missed due to stupid dependencies in the interface code... god, I'd like to clean this shit up better).